.gitattributes

```diff
@@ -0,0 +1,2 @@
+# Ignore lockfile diffs and text formatting
```

Comment:
Interesting I've never seen this done for lockfiles, I'm guessing this helps with merge conflicts?